features/app-launch-handler.yml

```
@@ -0,0 +1,9 @@
+name: Launch Handler
+description: The `launch_handler` manifest member is used to configure how web app launches should behave with respect to already open web app instances.
+spec: https://wicg.github.io/web-app-launch/#dfn-launch_handler
+group: progressive-web-app
+compat_features:
+  - api.LaunchQueue
+  - api.Window.launchQueue
+  - html.manifest.launch_handler
+  - html.manifest.launch_handler.client_mode
```

It seems to me that there are additional keys we could track here:

```suggestion
  - api.LaunchParams
  - api.LaunchParams.files
  - api.LaunchParams.targetURL
  - api.LaunchQueue
  - api.LaunchQueue.setConsumer
  - api.Window.launchQueue
  - html.manifest.launch_handler
  - html.manifest.launch_handler.client_mode
```

(Accepting this change means re-running `npm run dist` and committing the result.)